fv/hostendpoints_test.go

```diff
@@ -115,10 +115,69 @@ func describeHostEndpointTests(getInfra infrastructure.InfraFactory, allInterfac
 		cc.ExpectSome(felixes[0], w[0])
 		cc.ExpectSome(felixes[1], w[1])
 	}
+	expectHostToOwnPodViaServiceTraffic := func() {
+		// host to own pod always allowed, even via a service IP
+		for i := range felixes {
+			// Allocate a service IP.
+			serviceIP := fmt.Sprintf("10.96.0.%v", i+1)
+
+			// Add a NAT rule for the service IP.
+			felixes[i].Exec(
```

Comment:
Do you feel strongly that way?  I think I prefer the locality as it is now, and that it would be harder to understand with part of this moved to BeforeEach.

Reply:
Have introduced felix.ProgramIptablesDNAT to address the duplication here.